<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleDialog</class>
 <widget class="QDialog" name="BattleDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelHero">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMonster">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="HeroHealth">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="MonsterHealth">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFight">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сражаться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEvade">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбежать</string>
   </property>
  </widget>
  <widget class="QLabel" name="BattleInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>В комнате монстр. Можно попробовать сразиться</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
